### PR TITLE
fix: use send instead of send aliases

### LINKED ISSUE #1409: request: remove all references to send aliases (sendCode, sendFile[s], sendEmbed...)
**Is your feature request related to a problem? Please describe.**

To complete #1369's efforts, we would like to detach ourselves from Klasa's extensions and use structures where applicable, that way we have to go through one less effort in the rewrite to Sapphire.

**Describe the solution you'd like**

- Replace all references of `sendCode`, `sendEmbed`, `sendFile`, `sendFiles`, and `sendMessage` for their `send` counterparts. [`SendAliases`'s source code](https://github.com/skyra-project/klasa/blob/settings/src/extendables/SendAliases.js).
- Replace the permissions check getters with the underlying code, or move them to the [`SkyraTextChannel`](https://github.com/skyra-project/skyra/blob/main/src/lib/extensions/SkyraTextChannel.ts) extension. [`MessageExtendables`'s source code](https://github.com/skyra-project/klasa/blob/settings/src/extendables/MessageExtendables.js).

**Describe alternatives you've considered**

N/a.

**Additional context**

The convenience getters and methods will cease existing in Sapphire.

### PR DESCRIPTION
**Pull request by KunoichiZ**
~~Needs https://github.com/skyra-project/klasa/pull/5 to be merged first.~~ Merged.

### CODE REVIEW

File: src/commands/Fun/wakanda.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default class extends SkyraCommand {
 		const userAvatar = await fetchAvatar(user);
 		const image = this.generateImage(userAvatar);
 
-		return message.channel.sendFile(image.toBuffer(), 'we-do-not-do-that-here.png');
+		return message.channel.send(image.toBuffer(), { files: [{ attachment: image.toBuffer(), name: 'we-do-not-do-that-here.png' }] });
```
Comment (vladfrangu) [inline src/commands/Fun/wakanda.ts]: Invalid syntax, and is toBuffer async? Might wanna drop the first call

File: src/commands/Fun/peepolove.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default class extends SkyraCommand {
 			.printImage(this.handsImage, 0, 0, 512, 512)
 			.toBufferAsync();
 
-		return message.channel.sendFile(buffer, 'peepoLove.png');
+		return message.channel.send(buffer, { files: [{ attachment: buffer, name: 'peepoLove.png' }] });
```
Comment (vladfrangu) [inline src/commands/Fun/peepolove.ts]: Sending the buffer as content then the file? :3

File: src/commands/Google/weather.ts [hunk 3]
```diff
@@ -133,7 +133,7 @@ export default class extends SkyraCommand {
 
 			.toBufferAsync();
 
-		return message.channel.sendFile(attachment, `${geoCodeLocation}.png`);
+		return message.channel.send(attachment, { files: [{ attachment, name: `${geoCodeLocation}.png` }] });
```
Comment (vladfrangu) [inline src/commands/Google/weather.ts]: sending buffer and option?

File: src/commands/Misc/triggered.ts [hunk 4]
```diff
@@ -33,7 +33,7 @@ export default class extends SkyraCommand {
 
 	public async run(message: KlasaMessage, [user = message.author]: [User]) {
 		const attachment = await this.generate(user);
-		return message.channel.sendFile(attachment, 'triggered.gif');
+		return message.channel.send(attachment, { files: [{ attachment, name: 'triggered.gif' }] });
```
Comment (vladfrangu) [inline src/commands/Misc/triggered.ts]: ditto

File: src/commands/Moderation/Utilities/permissions.ts [hunk 5]
```diff
@@ -44,6 +44,6 @@ export default class extends SkyraCommand {
 			.setTitle(language.get(LanguageKeys.Commands.Moderation.Permissions, { username: user.tag, id: user.id }))
 			.setDescription(list.join('\n'));
 
-		return message.sendMessage({ embed });
+		return message.send({ embed });
```
Comment (vladfrangu) [inline src/commands/Moderation/Utilities/permissions.ts]: no need for object syntax

File: src/commands/Tools/Websearch/dsearch.ts [hunk 6]
```diff
@@ -30,7 +30,7 @@ export default class extends SkyraCommand {
 			embed.addField(await message.fetchLocale(LanguageKeys.Commands.Tools.DuckDuckGoLookalso), body.RelatedTopics[0].Text);
 		}
 
-		return message.sendMessage({ embed });
+		return message.send({ embed });
```
Comment (vladfrangu) [inline src/commands/Tools/Websearch/dsearch.ts]: d

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Review by kyranet — APPROVED** [review]

**Review by favna — APPROVED** [review]

**Pull request merged**